pages/common/rbt.md

```diff
@@ -0,0 +1,24 @@
+# rbt
+
+> RBTools is a set of command line tools for working with Review Board and RBCommons.
+> Homepage: <https://www.reviewboard.org/docs/rbtools/dev/>.
+
+- Post changes to Review Board:
+
+`rbt post {{changenum}}`
+
+- Display the diff that will be sent to Review Board:
+
+`rbt diff`
+
+- Land a change in a local branch or on a review request:
+
+`rbt land {{branchname}}`
+
+- Patche your tree with a change on a review request:
+
+`rbt patch {{reviewrequestid}}`
+
+- Set up RBTool to talk to your repository:
```

Comment:
Minor nit: we try to avoid having direct speech in our pages. Please replace your with one's or anything else you feel like.

Reply:
I'd suggest:

> Set up RBTool to talk to a repository:

Reply:
Changed